.github/workflows/test.yml

```diff
@@ -1,6 +1,10 @@
 name: Test
 
-on: [push, pull_request]
```

Comment:
This is currently causing a duplicate run on every PR, eg.

![Screen Shot 2023-05-31 at 12 39 26 PM](https://github.com/WordPress/wporg-two-factor/assets/1017872/2c109999-001c-42ef-9e61-3087729ae964)

Reply:
Use this format:
```
    push:
        branches: [trunk]
    pull_request:
```
See: https://github.com/WordPress/wporg-main-2022/blob/trunk/.github/workflows/linters.yml

Reply:
Yep, see change below 😄